docs/index.md

```diff
@@ -163,6 +163,50 @@ pages:
   - Home: index.md
   - About: about.md</code></pre>
 
+
+### Extending Cinder
+
+Create a new directory within your project (e.g., `cinder-theme-ext/`) and create `main.html`. Add the following line at the top of the HTML file.
+
+```django hljs
+{% extends "base.html" %}
+```
+
+Instead of using `theme: cinder` or `theme_dir: cinder` in `mkdocs.yml`, use:
+```yaml
+theme:
+      name: cinder
+      custom_dir: [custom dir]
+```
+
+Refer to [MkDocs Documentation - Using the theme custom_dir](https://www.mkdocs.org/user-guide/styling-your-docs/#using-the-theme-custom_dir) for more information.
+
+#### Adding extra HTML to the head tag
+Append to `main.html`:
+```django hljs
+{% block extrahead %}
+      <meta name="author" content="{{ page.meta.author }}">
+{% endblock %}
+```
+
+#### Replacing footer
+Append to `main.html`:
+```django hljs
+{% block footer %}
+<hr>
+<p>{% if config.copyright %}
+      <small>{{ config.copyright }}<br></small>
+{% endif %}
+<small>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a> and <a href="https://sourcefoundry.org/cinder/">Cinder theme.</a></small>
+{% if page.meta.revision_date %}
```

Comment:
@zhaoterryy 

Terry can you comment on this block of code? Is this intended to support your extension?

Reply:
Ah yes.. I just did a quick copy and paste. We could remove the offending lines or make it something more generic like `config.build_date_utc`? What are your thoughts on this?

Reply:
I checked out your extension and this would be a nice feature to support if you are interested in doing so.  If you are willing to author documentation to explain to users how it is implemented I say we leave it.  Without docs I say we remove it.  Thoughts?

Reply:
https://github.com/chrissimpkins/cinder/pull/57